### ISSUE TITLE
BUG: log_viewers cognito pool recreated on account-specific terraform run

### ISSUE THREAD

**Issue by adunkman** [ustaxcourt/ef-cms#496]
The `log_viewers` cognito pool is modified every time the account-specific terraform is run, showing this difference: 

```
  # aws_cognito_identity_pool.log_viewers will be updated in-place
  ~ resource "aws_cognito_identity_pool" "log_viewers" {
        allow_unauthenticated_identities = false
        arn                              = "arn:aws:cognito-identity:us-east-1:350455059537:identitypool/us-east-1:55e8d40b-2580-4f6f-a710-a4e074270133"
        id                               = "us-east-1:55e8d40b-2580-4f6f-a710-a4e074270133"
        identity_pool_name               = "kibana dashboard identity pool"
        openid_connect_provider_arns     = []
        saml_provider_arns               = []
        supported_login_providers        = {}
        tags                             = {}

      - cognito_identity_providers {
          - client_id               = "641p536mtf742ev6s24q9604d" -> null
          - provider_name           = "cognito-idp.us-east-1.amazonaws.com/us-east-1_qJr5ZVWNT" -> null
          - server_side_token_check = true -> null
        }
    }
```

It appears that we need to specify the configuration for the `cognito_identity_providers` block in terraform. This looks like configuration that’s auto-created when applying the pool to Kibana.

**Comment by adunkman** [comment]
Discovered this is actually an issue causing intermittent lockouts of Kibana — the cognito identity providers are being automagically applied by AWS Elasticsearch and then un-applied by terraform. Pulling in a fix to #532!

**Comment by adunkman** [comment]
Fixed in #532.
